platform-tests/src/test/java/org/junit/platform/console/ConsoleLauncherIntegrationTests.java

```diff
@@ -32,16 +34,23 @@ void executeWithoutArgumentsFailsAndPrintsHelpInformation() {
 		);
 	}
 
-	@Test
-	void executeWithoutExcludeClassnameOptionDoesNotExcludeClassesAndMustIncludeAllClassesMatchingTheStandardClassnamePattern() {
-		String[] args = { "-e", "junit-jupiter", "-p", "org.junit.platform.console.subpackage" };
+	@ParameterizedTest
+	@ValueSource(strings = { "-e junit-jupiter -p org.junit.platform.console.subpackage",
+			"execute -e junit-jupiter -p org.junit.platform.console.subpackage" })
+	void executeWithoutExcludeClassnameOptionDoesNotExcludeClassesAndMustIncludeAllClassesMatchingTheStandardClassnamePattern(
```

Comment:
Good candidate for longest method name in this project. ;-)

Reply:
I had to rotate my screen to read it in full ;)

Reply:
> Good candidate for longest method name in this project. ;-)

Mmmm.... looks like I'm getting some competition! 😈